optuna/integration/sklearn.py

```diff
@@ -56,6 +57,24 @@
 _logger = logging.get_logger(__name__)
 
 
+def check_fit_params(
+    X,  # type: TwoDimArrayLikeType
+    fit_params,  # type: Dict
+    indices  # type: OneDimArrayLikeType
+):
+    assert _available
```

Comment:
This line is redundant as the availability check is done before this function is called. ref: https://github.com/optuna/optuna/blob/4a5466cf089791d49dfc99e836d80ced011ac75e/optuna/integration/sklearn.py#L698
```suggestion

```